docs/release_notes/2024/nov2024.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+id: November_2024
+description: Release notes for November
+title: November 2024
+tags:
+  - release notes
+  - changelog
+  - november
+---
+
+## 3.4.1.\* (November 25, 2024)
+
+- Prophecy Python libs version: 1.9.24
+- Prophecy Scala libs version: 8.4.0
+
+### Features {#Features341}
```

Comment:
are the curly braces here used to ID the heading for link reference? just curious

Reply:
I can be used for reference, but we're using them here mainly to differentiate them from future ### Features headings. So if there were another release notes in November, then it would use ### Features {#Features342}. (But we most likely won't have another release this month.)

Reply:
Great, exactly what I was looking for!